blocks/footer-block/features/footer/default.jsx

```diff
@@ -6,21 +6,16 @@ import { useFusionContext } from 'fusion:context';
 import getProperties from 'fusion:properties';
 import getThemeStyle from 'fusion:themes';
 import Link from '@wpmedia/links-bar-block';
-
 import {
   FacebookAltIcon,
   TwitterIcon,
   RssIcon,
   LazyLoad,
   isServerSide,
 } from '@wpmedia/engine-theme-sdk';
-
+import { PrimaryFont } from '@wpmedia/shared-styles';
```

Comment:
can you add shared styles as a peerDep of footer block. any block that imports shared styles should have that package as a peer dep

Reply:
yeah i just missed a few of them, ill add that